backend-docker/docker-compose.yaml

```diff
@@ -2,14 +2,15 @@ version: "3"
 
 services:
   tmail-backend:
-    image: linagora/tmail-backend:memory-1.0.0
+    image: linagora/tmail-backend:memory-1.0.1-rc1
```

Comment:
we are using `linagora/tmail-backend:memory-1.0.3-rc1`

Reply:
updated